esp/services/ws_esdlconfig/ws_esdlconfigservice.cpp

```diff
@@ -157,10 +157,48 @@ bool isESDLDefinitionBound(const char * esdldefname, int version)
     return isESDLDefinitionBound(id);
 }
 
+
+void fetchESDLDefinitionFromDaliByNameOnly(const char * name, StringBuffer & def)
+{
+    if (!name || !*name)
+        throw MakeStringException(-1, "Unable to fetch ESDL Service definition information, definition name is not available");
```

Comment:
Good point, as of now this is pervasive in this class and others, a dedicated task should be opened to tackle this issue.